### PR TITLE
Fix for tiny MCE when language is en-US

### PR DESCRIPTION
**Pull request by RachBreeze**
### Prerequisites
When opening a document type with the tinyMCE on it users are warned about a missing en-US tinyMCE file.  This PR resolves that

### CODE REVIEW

File: src/Umbraco.Web.UI.Client/lib/tinymce/langs/en_US.js [hunk 1]
```diff
@@ -0,0 +1,261 @@
+tinymce.addI18n('en_CA',{
```
Comment (bjarnef) [inline src/Umbraco.Web.UI.Client/lib/tinymce/langs/en_US.js:1]: @RachBreeze I think this should be `en_US`.
We already have `en_CA` language here: https://github.com/umbraco/Umbraco-CMS/blob/v8/contrib/src/Umbraco.Web.UI.Client/lib/tinymce/langs/en_CA.js

Comment (RachBreeze) [reply src/Umbraco.Web.UI.Client/lib/tinymce/langs/en_US.js:1]: Hi @bjarnef 

Oof yes :-) good spot thank you

**Comment by emmaburstow** [comment]
Hola Rachel,

Thanks for this work. Lovely to see you and Bjarne collaborating too. We'll shout if we need you,

Emma

File: src/Umbraco.Web.UI.Client/lib/tinymce/langs/en_US.js (hunk 1 above)
Comment (bjarnef) [reply src/Umbraco.Web.UI.Client/lib/tinymce/langs/en_US.js:1]: @RachBreeze did you gad any time to update the PR? 😎👍

**Comment by RachBreeze** [comment]
Many thanks @bjarnef  for fixing

**Comment by nul800sebastiaan** [comment]
Thanks @RachBreeze! 👍

**Pull request merged**